sysc3110 project/src/model/PeaShooterTest.java

```diff
@@ -0,0 +1,35 @@
+package model;
+
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PeaShooterTest {
+	PeaShooter p;
+
+	@Before
+	public void setUp() throws Exception {
+		p = new PeaShooter();
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void testPeaShooter() {
+		assert (p.getType() == ModelType.PEA_SHOOTER);
```

Comment:
If we add asserts for things like `getAtk()` then there will be so much refactoring if we change these default values.